### ISSUE TITLE
False positive for Void redundant_void_return

### ISSUE THREAD

**Issue by ryanmeisters** [realm/SwiftLint#1382]
Something like
```
private func makeActionHandler(...) -> VoidClosure {
        return { [unowned self] in
              .... 
        }
}
```
gives `Redundant Void Return Violation: Returning Void in a function declaration is redundant. (redundant_void_return)`

Seems like the rule is just matching anything with Void in it.. should be an easy enough fix. I will submit a PR if I can

**Comment by marcelofabri** [comment]
This is already fixed on master.
